pyqtgraph/graphicsItems/PlotCurveItem.py

```diff
@@ -784,7 +783,8 @@ def paint(self, p, opt, widget):
 
         if do_fill:
             if self._shouldUseFillPathList():
-                paths = self._getFillPathList()
+                chunksize = 50 if not isinstance(widget, QtWidgets.QOpenGLWidget) else 5000
```

Comment:
I think this line should be moved to the `_getFillPathList` method and remove the `chunksize` argument to the function.  Also probably should have a comment on how the values of 50 and 5000 were tested and determined to be ideal values for their respective use cases.